models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -350,7 +382,7 @@ protected function convertToWebPath(array $pathReference): ?string
      */
     public function getFrontendPath(): string
     {
-        $path = $this->getPath();
+        $path = $this->getPath(['forceFrontend' => true, 'deferredAllowed' => false]);
```

Comment:
```suggestion
        $path = $this->getPath(['deferredAllowed' => false, 'forceFrontend' => true]);
```
all these are not relevant suggestion, but looks more in line with all the variables order